.github/ISSUE_TEMPLATE/config.yml

```
@@ -1,7 +1,10 @@
-blank_issues_enabled: true
+blank_issues_enabled: false
 contact_links:
   - name: Usage question
-    url: https://stackoverflow.com/questions/tagged/python-xarray
-    about: "Post a question on Stack Overflow using the #python-xarray
-    tag. These are regularly reviewed by xarray's maintainers, and questions which
-    include a reproducible example will receive a response."
\ No newline at end of file
+    url: https://github.com/pydata/xarray/discussions
+    about: |
+      Ask questions and discuss with other community members here.
+      If you have a question like `How do I concatenate a list of datasets?` then 
```

it seems `about` is a plain text description, so we unfortunately can't use markdown here:
```suggestion
      If you have a question like "How do I concatenate a list of datasets?" then
```

---

👍🏽